kerlescan/view_helpers.py

```diff
@@ -127,3 +129,31 @@ def log_username(logger, request):
             )
         else:
             logger.debug("identity header not sent for request")
+
+
+def validate_uuids(system_ids):
+    """
+    helper method to test if a UUID is properly formatted. Will raise an
+    exception if the format is wrong.
+    """
+    malformed_ids = []
+    for system_id in system_ids:
+        # the UUID() check was missing some characters, so adding regex first
+        if not (
+            re.match(
+                r"^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$",
+                system_id.lower(),
+                re.ASCII,
```

Comment:
I think you can skip definining this, since you aren't using `\w` anymore.

from the doc:

Make \w, \W, \b, \B, \d, \D, \s and \S perform ASCII-only matching instead of full Unicode matching. This is only meaningful for Unicode patterns, and is ignored for byte patterns. Corresponds to the inline flag (?a).